### DISCUSSION TITLE
Welcome to github.dart Discussions!

### DISCUSSION THREAD

**Discussion by robrbecker** [SpinlockLabs/github.dart#239] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yahu1031** [comment]
Do we have any discord server for this?? May for support💪 or any help, feedback? Not gitter cause not may use it.

**Reply by robrbecker** [reply]
There is not a discord. There is very little discussion so I figured the github built-in discussions would be sufficient.

**Reply by yahu1031** [reply]
Thanks for the reply @robrbecker 

https://pub.dev/documentation/github/latest/github/Authentication/Authentication.withToken.html  can you please tell me what is the token I should pass and how do i get it?? Is that authsecret token I get in developers tool with client id in github?

**Reply by robrbecker** [reply]
Try https://help.github.com/articles/creating-an-access-token-for-command-line-use
